web/src/stream_ui_updates.js

```diff
@@ -448,7 +448,11 @@ export function enable_or_disable_add_subscribers_elements(
         const $add_subscribers_button = $container_elem
             .find('button[name="add_subscriber"]')
             .expectOne();
-        $add_subscribers_button.prop("disabled", !enable_elem);
+
+        if ($input_element.text().length === 0) {
+            $add_subscribers_button.prop("disabled", true);
+        }
```

Comment:
It's not clear from the commit message why this change is being made. Please explain the issue the commit is solving in detail in the commit description. Also explain why this change solves that issue and why we would not have regressions after this change.

It would also be good to add a comment here while we're at it.

Reply:
Added a detailed commit message:

In channel settings > subscribers tab, the `Add` button should
stay disabled when add_subscriber input is empty.
`enable_or_disable_add_subscribers_elements` function enables the
input even if it is empty. This feels like jitter.

This PR fixes this behavior by disabling the `Add` button when input is
empty.

Reply:
From an abstractions standpoint, I feel like `enable_elem` should be being calculated differently, not that we should do something other than using that value. Is there a reason you didn't take that approach?

Reply:
`enable_or_disable_add_subscribers_elements` enables/disables the whole input:

![image](https://github.com/user-attachments/assets/0bff1d05-3a12-468f-b866-6111b9ac1d2d)

When the input is enabled (usually) (i.e. `enable_elem` is true), the add button is getting enabled even when it is empty. Thus it is being calculated correctly.

The issue becomes clear here:

![add_button_issue](https://github.com/user-attachments/assets/763bf27e-ac97-4ba7-a3de-40b038c7c49b)

The add button gets enabled, when we type something and remove it, it becomes disabled since input is empty.